src/index.html

```diff
@@ -9,6 +9,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <container class="container">
+        <a class="img" href="#home">
+          <img
+          src="/images/logo.png"
+          alt="MOYO logo"
+          class="logo"
+          >
+        </a>
```

Comment:
```suggestion
<img
  src="/images/logo.png"
  alt="MOYO logo"
  class="logo"
>
```
Consider fixing indentations accordingly to the code style guide